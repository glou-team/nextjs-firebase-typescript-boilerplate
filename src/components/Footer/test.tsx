import { render, screen } from '@testing-library/react'

import Footer from '.'

describe('<Footer />', () => {
  it('Should render the footer', () => {
    const { container } = render(<Footer />)

    expect(screen.getByText(/Powered by/i)).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
