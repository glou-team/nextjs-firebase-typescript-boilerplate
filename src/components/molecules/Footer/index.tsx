import * as S from './styles'

const Footer = ({ message = '' }) => {
  return <S.Container>{message}</S.Container>
}

export default Footer
