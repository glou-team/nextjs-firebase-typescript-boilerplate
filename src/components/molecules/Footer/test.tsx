import { render } from '@testing-library/react'

import Footer from '.'

describe('<Footer />', () => {
  it('Should render the footer', () => {
    const { container } = render(<Footer />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
