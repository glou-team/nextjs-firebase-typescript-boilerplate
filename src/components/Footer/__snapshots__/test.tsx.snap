// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Should render the footer 1`] = `
<footer
  class="jsx-920434584"
>
  Powered by
  <a
    class="jsx-920434584"
    href="https://firebase.google.com/products/hosting"
    rel="noopener noreferrer"
    target="_blank"
  >
    <img
      alt="Firebase Hosting Logo"
      class="jsx-920434584"
      src="/firebase-hosting.svg"
    />
  </a>
  &
  <a
    class="jsx-920434584"
    href="https://zeit.co?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
    rel="noopener noreferrer"
    target="_blank"
  >
    <img
      alt="Next.js Logo"
      class="jsx-920434584 nextjs"
      src="/nextjs.svg"
    />
  </a>
</footer>
`
